admin-dev/themes/default/js/calendar.js

```diff
@@ -380,7 +380,7 @@ $( document ).ready(function() {
 	});
 
 	$('#datepicker-compare').click(function() {
-		if ($(this).attr("checked")) {
+    if ($(this).is(":checked")) {
```

Comment:
```suggestion
    if ($(this).prop('checked')) {
```
Should be better, no?